test/features/users.feature

```diff
@@ -137,13 +137,9 @@ Feature: User APIs
         And I go to the dashboard
         Then I should see an element with xpath "//h2[contains(string(), 'News feed')]"
 
-    Scenario: Dashboard page is not accessible anonymously
-        When I go to the dashboard
-        Then I should see an element with xpath "//*[contains(string(), 'Not authorised to see this page')]"
-
 
     Scenario: Password reset works
-        When I request a password reset for "publisher"
+        When I request a password reset for "editor"
```

Comment:
can you add a test for test change password with special key word block based on user_creation.feature 

can you also add tag's for these tests which are related to the username/display name validation.

Reply:
Can do.

Reply:
The relevant tests are already in `user_creation.feature` and are tagged `@user_creation`.